atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/TracingPrefsConfig.java

```diff
@@ -44,9 +44,11 @@
     public void run() {
         try {
             final File TRACING_PREF_FILE = new File(System.getProperty("user.dir") + java.io.File.separatorChar + TRACING_PREF_FILENAME);
+
             if (TRACING_PREF_FILE.exists()) {
+                FileInputStream fileStream = new FileInputStream(TRACING_PREF_FILE);
                 try {
```

Comment:
iirc there's one or two commons-y or atlasdb-client-y packages that workspace still relies on (java 6)

... I'll just try it and see if it works.